packages/fast-components-react-base/src/media/DOCS.MD

```diff
@@ -0,0 +1,11 @@
+## Best practices
+Use *image* to break up text-heavy pages, to illustrate products for sale, or to improve storytelling on a page. The use of *image* should always support the page’s story and, when possible, contribute to that story. Size the *image* to complement the rest of the page content. Use the **picture** option to employ a selection of image versions for different viewport sizes.
+
+### Usage guidance
+Use the [Lifestyle](/getting-started/designing/imagery-lifestyle) and [Product](/getting-started/designing/imagery-product) imagery guidance when choosing your images. Pay attention to guidance for improper usage at the bottom of the Lifestyle and Product imagery pages. Use HTML text instead of putting graphic text in images.

```

Comment:
This will be interesting. @awentzel - this is very specific to MSFT. I think we should break these up between base vs. msft if we have to.

Reply:
I figured as much just wanted to get this in there to lean on you guys for direction

Reply:
I would actually replace that "Usage guidance" with what is being said on the page it's referring too.  For example, https://fluentweb.com/designing/imagery/lifestyle from the art direction, do's and don'ts section.  Look to consolidate where you can, and remove anything specific to Fluent Web or MSFT.

Reply:
Now, @chrisdholt, are you saying that maybe the entire perspective around Lifestyle vs Product is a MSFT thing?  If so, then sure yeah, we could probably eliminate that and just keep it to generically web practices for applying images or pictures.

Reply:
@awentzel - yeah, that seems like our design language guidance

Reply:
We can pull this out then.  Move the design language guidance into MSFT styles project as `.md` and then keep whatever is relevant for base shape/accessibility in here.